examples/sorting_cpp/quickSort.cpp

```diff
@@ -0,0 +1,127 @@
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <string>
+
+int block_size = 16;
+void (*block_algo)(std::vector<int>&, int, int);
```

Comment:
Naming is inconsistent here. I think we should use either `camelCase` or `snake_case` everywhere.